tests/environment-hook.bats

```diff
@@ -13,14 +13,17 @@ load '/usr/local/lib/bats/load.bash'
   export BUILDKITE_PLUGIN_VAULT_SECRETS_SERVER=https://vault_svr_url
   export BUILDKITE_PLUGIN_VAULT_SECRETS_DUMP_ENV=true
   export BUILDKITE_PIPELINE_SLUG=testpipe
-  export TESTDATA=`echo MY_SECRET=fooblah | base64`
+  export TESTDATA='MY_SECRET=fooblah'
+  export BUILDKITE_PLUGIN_VAULT_SECRETS_AUTH_SECRET_ENV="abcde12345"
+  export BUILDKITE_PLUGIN_VAULT_SECRETS_AUTH_ROLE_ID=buildkite
+  export BUILDKITE_PLUGIN_VAULT_SECRETS_AUTH_METHOD=approle
 
   stub vault \
-    'auth -address=https://vault_svr_url - : echo Successfully authenticated. You are now logged in' \
-    "list -address=https://vault_svr_url -format=yaml data/buildkite/testpipe : exit 0 " \
-    "list -address=https://vault_svr_url -format=yaml data/buildkite : echo -e '- env'" \
-    "list -address=https://vault_svr_url -format=yaml data/buildkite/env : echo -e '- MY_SECRET'" \
-    "read -address=https://vault_svr_url -field=value data/buildkite/env/MY_SECRET : echo ${TESTDATA}"
+    "write -field=token -address=https://vault_svr_url auth/approle/login role_id=buildkite secret_id=\"${BUILDKITE_PLUGIN_VAULT_SECRETS_AUTH_SECRET_ENV}\" : echo 'Successfully authenticated with Role ID ${BUILDKITE_PLUGIN_VAULT_SECRETS_AUTH_ROLE_ID}'" \
+    "kv list -address=https://vault_svr_url -format=yaml data/buildkite/testpipe : echo 'env'" \
+    "kv list -address=https://vault_svr_url -format=yaml data/buildkite : echo 'env'" \
+    "kv get -address=https://vault_svr_url -field=data -format=yaml data/buildkite/testpipe/env : echo ${TESTDATA}" \
+    "kv get -address=https://vault_svr_url -field=data -format=yaml data/buildkite/env : echo ${TESTDATA}"
```

Comment:
You are outputting the same string twice, how can you distinguish them in the output? (i.e.: if one of these is not called, the `assert_output` would not fail). I understand that other assertions or the unstub most likely would, but if you are testing some functionality, it may be a good idea to test it all.